<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateNewUnit_Ask</class>
 <widget class="QWidget" name="CreateNewUnit_Ask">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>593</width>
    <height>310</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>593</width>
    <height>310</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1200</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>HRD ver 1.0</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background:url(:resources/img/unit.png);
background-size: cover;</string>
  </property>
  <widget class="QLabel" name="emrg_old_unit">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>40</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>591</width>
     <height>301</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="enter_unit_where">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">background:transparent</string>
            </property>
            <property name="text">
             <string>Введите отдел, в который добавить работника</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="combo_old_unit">
            <property name="styleSheet">
             <string notr="true">background:white</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="enter_name_worker">
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">background:transparent</string>
              </property>
              <property name="text">
               <string>Имя работника</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="line_name">
              <property name="styleSheet">
               <string notr="true">background:white</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label">
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">background:transparent</string>
              </property>
              <property name="text">
               <string>Год рождения</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="spin_year">
              <property name="styleSheet">
               <string notr="true">background:white</string>
              </property>
              <property name="minimum">
               <number>1800</number>
              </property>
              <property name="maximum">
               <number>2019</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">background:transparent</string>
              </property>
              <property name="text">
               <string>Должность</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="line_post">
              <property name="styleSheet">
               <string notr="true">background:white</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">background:transparent</string>
              </property>
              <property name="text">
               <string>Образование</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="line_educ">
              <property name="styleSheet">
               <string notr="true">background:white</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="label_4">
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">background:transparent</string>
              </property>
              <property name="text">
               <string>Зарплата</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="spin_salary">
              <property name="styleSheet">
               <string notr="true">background:white</string>
              </property>
              <property name="maximum">
               <number>1000000</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QRadioButton" name="create_unit">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">background:transparent</string>
            </property>
            <property name="text">
             <string>Создать новый отдел</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <widget class="QLabel" name="new_unit">
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">background:transparent</string>
              </property>
              <property name="text">
               <string>Название нового отдела</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="line_new_unit">
              <property name="styleSheet">
               <string notr="true">background:white</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QRadioButton" name="not_create_unit">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background:transparent</string>
          </property>
          <property name="text">
           <string>Не создавать
 новый отдел</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label_emrg_create">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background:transparent</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="add_worker">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>оформить назначение</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
